Project.toml

```
@@ -73,7 +73,7 @@ Requires = "1.1"
 SciMLBase = "1.90"
 Setfield = "0.8, 1"
 SimpleUnPack = "1.1"
-StartUpDG = "0.16"
+StartUpDG = "0.17"
```

```suggestion
StartUpDG = "0.16, 0.17"
```
Then we should be able to use this, shouldn't we? But maybe it's just fine to require an upgrade